packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.13.2
+
+* Adds `decidePolicyForNavigationResponse` to internal WKNavigationDelegate to support the
```

Comment:
This is copy/pasted from a previous changelog entry, and is unrelated to the PR.